ts/Extensions/Annotations/Popup.ts

```diff
@@ -1173,7 +1413,7 @@ H.Popup.prototype = {
                             parentFullName,
                             type,
                             parentDiv,
-                            ''
+                            '' as any
```

Comment:
Can we get rid of this `as any`?

Reply:
It's not my part of the code.
I've only specified the type which the `addInput` method should take but for some reason in some cases, it's required to be an empty string that is why I casted it to `any`.

Reply:
Perhaps it should be an empty object? 🤔 
It seems `''` is leveraging undefined references later: `emptyString.value`/`htmlFor`/`labelClassName` etc resolve to `undefined`